_foundations/typography/typeface-guidance.md

```diff
@@ -3,11 +3,12 @@
   <div class="accordion-panel">
     <ul>
       <li>Use just 1 sans serif font (Open Sans). Open Sans is a sans-serif font selected for accessibility and readability.</li>
-      <li>Use monospace... [PASCAL TO FILL IN & CSS FOR FONT SPECIMEN PLEASE]</li>
+      <li>When using the SASS source refer to the <code>$base-sans-serif</code> variable to access the sans-serif font-stack.</li>
```

Comment:
But actually can we just say `Use the <code>$base-sans-serif</code> variable to use the sans-serif font-stack.` <-- what's the reason for specifying Sass? Is there potential confusion in not mentioning?